app/services/oauth2_identity_api.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+class Oauth2IdentityApi < IdentityApi
```

Comment:
How about this name?

```suggestion
class IdentityUsersApi < IdentityApi
```